.goreleaser.yml

```diff
@@ -0,0 +1,29 @@
+before:
+  hooks:
+    - go vet ./...
+    - go test ./...
+    - go mod download
+    - go generate ./...
+builds:
+- env:
+  - CGO_ENABLED=0
+  - GO111MODULE=on
+  ldflags:
+    - -s -w -X github.com/mittwald/spacectl/buildinfo.Version={{.Tag}} -X github.com/mittwald/spacectl/buildinfo.Hash={{.ShortCommit}} -X github.com/mittwald/spacectl/buildinfo.BuildDate={{.Date}}
+archive:
+  replacements:
+    darwin: Darwin
```

Comment:
Up until now, we've always used lowercased OS names in the release files. Are these replacements even necessary?